app/src/main/java/ch/epfl/sdp/fragment/UserInfectionFragment.java

```diff
@@ -166,24 +175,41 @@ private void getLoggedInUser() {
     private void executeHealthStatusChange() {
         CharSequence buttonText = infectionStatusButton.getText();
         boolean infected = buttonText.equals(getResources().getString(R.string.i_am_infected));
-        if (infected) {
-            //Tell the analyst we are now sick !
-            service.getAnalyst().updateStatus(Carrier.InfectionStatus.INFECTED);
-            setInfectionColorAndMessage(true);
-            modifyUserInfectionStatus(userName, true,
-                    value -> {
-                        //infectionUploadView.setText(String.format("%s at %s", value, Calendar.getInstance().getTime()));
-                    });
-        } else {
-            //Tell analyst we are now healthy !
-            service.getAnalyst().updateStatus(Carrier.InfectionStatus.HEALTHY);
-            sendRecoveryToFirebase();
-            setInfectionColorAndMessage(false);
-            modifyUserInfectionStatus(userName, false,
-                    value -> {
-                        //infectionUploadView.setText(String.format("%s at %s", value, Calendar.getInstance().getTime()))
-                    });
+
+        Date currentTime = Calendar.getInstance().getTime();
+        /* get 1 jan 1970 by default. It's definitely wrong but works as we want t check that
```

Comment:
Why is it wrong ? every time that we have on our app is relative to that famous 1 jan 1970.

Reply:
It's the epoch, the beginning of time for unix people. It is very similar to the birth of Jesus for Christians.

Reply:
Yes ahah. But my quesiton swas about your comment : " It's definitely wrong " - nevermind.